kahuna/public/js/preview/image.html

```diff
@@ -75,18 +75,20 @@
                 {{::ctrl.image.data.uploadTime | date:'HH:mm'}}
             </span>
 
-            <span class="bottom-bar__meta-item titip-bottom preview__in-collection"
-                ng:if="ctrl.image.data.collections.length > 0">
-                <gr-icon gr-small>folder</gr-icon>
-                <span class="titip-content preview__in-collection__list">
-                    <li ng-repeat="collection in ctrl.image.data.collections">
-                        <a ui:sref="search.results({query: '~'+collection.data.pathId})">
-                            {{collection.data.path.join(' ▸ ')}}
+            <ul ng:repeat="collectionGroup in ctrl.image.data.collectionGroups"
+                class="bottom-bar__meta-item titip-bottom preview__collections">
+                <li class="preview__collections__collection" style="background-color: {{collectionGroup.cssColour}}">
+                    {{collectionGroup.basename}}
+                </li>
+                <ul class="titip-content preview__collections__list">
```

Comment:
Don't think a `ul` can have a `ul` as a child, it should be wrapped in an `li` ([reference](https://developer.mozilla.org/en/docs/Web/HTML/Element/ul)).